test/index_test.js

```diff
@@ -44,6 +45,11 @@ describe('maskXml()', () => {
     `);
   });
 
+  it('should skip masking non-strings', () => {
+    should.not.exist(maskXml(['foo', 'bar'])(undefined));
```

Comment:
How about this (just a readability preference):

```
[undefined, { biz: 'baz' }].forEach(value => {
  maskXml(['foo', 'bar'])(value).should.eql(value);
})
```

Reply:
Fails with a `TypeError: Cannot read property 'should' of undefined`, but this works:
```
    [undefined, { biz: 'baz' }].forEach(value => {
      should(maskXml(['foo', 'bar'])(value)).eql(value);
    });
```

Reply:
Fixed. Included other minor improvements too.